samcli/local/docker/manager.py

```diff
@@ -124,3 +192,10 @@ def has_image(self, image_name):
 
 class DockerImagePullFailedException(Exception):
     pass
+
+
+class DockerContainerException(UserException):
```

Comment:
We have a convention of not using the `UserException` inside the library classes. `UserException` is strictly a CLI interface exception. Can you instead catch the `DockerContainerException` inside the `do_cli` methods and re-raise as a UserException?

This will allow us to move the library into a separate package away from the CLI interface.

Reply:
Makes sense, I will fix that